src/components/Layout/Header/GlobalSearch/GlobalSearchButton.tsx

```diff
@@ -44,6 +48,8 @@ import { StakingResults } from './StakingResults/StakingResults'
 import { TxResults } from './TxResults/TxResults'
 import { makeOpportunityRouteDetails } from './utils'
 
+type ResultsWithActions = [...SelectGlobalItemsFromFilterReturn, SendResult[]]
```

Comment:
Variadic tuple needed because of the strong typing and the concat, we need an array with all items including send (action) items, which need to be fetched async so we unfortunately can't have them live in selector-land